_data/simple-icons.json

```diff
@@ -649,6 +649,20 @@
                 ]
             }
         },
+        {
+            "title": "Amazon SES",
+            "hex": "DD344C",
+            "source": "https://aws.amazon.com/architecture/icons/",
+            "guidelines": "https://aws.amazon.com/architecture/icons/",
+            "aliases": {
+                "aka": [
+                    "Amazon Simple Email Service",
+                    "Amazon SES",
+                    "AWS Simple Email Service",
+                    "AWS SES"
+                ]
```

Comment:
I'm not sure we need all/any of these aliases. Perhaps have AWS SES as the abbreviation, and Amazon Simple Email Service as the full title?